diff.txt

```diff
@@ -0,0 +1,879 @@
+directory 'bndtools/'
+directory 'bndtools/'
+directory 'bndtools/'
+
+
+blade.authenticator.shiro-1.0.0.jar
+*******************************************************
+
+
+blade.authfailure-1.0.0.jar
+*******************************************************
+
+
+blade.configurationaction-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+CHANGED    RESOURCES  <resources>
+ ADDED      RESOURCE   OSGI-INF/metatype/com.liferay.blade.samples.configurationaction.MessageDisplayConfiguration.xml
+
+
+blade.corejsphook-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ CHANGED    HEADER     Import-Package
+  ADDED      CLAUSE     javax.servlet
+  ADDED      CLAUSE     javax.servlet.http
+
+
+blade.friendlyurl-1.0.0.jar
+*******************************************************
+
+
+blade.gogo-1.0.0.jar
+*******************************************************
+
+
+blade.hook.jsp-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Import-Package
+
+
+blade.hook.resourcebundle-1.0.0.jar
+*******************************************************
+
+
+blade.indexerpostprocessor-1.0.0.jar
+*******************************************************
+
+
+blade.jsf.portlet.ds-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ CHANGED    HEADER     Import-Package
+  CHANGED    CLAUSE     javax.servlet
+   ADDED      PARAMETER  version:[3.0,4)
```

Comment:
we're using defaults everywhere.

On Fri, Apr 15, 2016 at 3:33 PM, Gregory Amerson <notifications@github.com>
wrote:

> In diff.txt
> <https://github.com/gamerson/liferay-blade-samples/pull/12#discussion_r59929143>
> :
>
> > +
> > +
> > +blade.hook.resourcebundle-1.0.0.jar
> > +*******************************************************
> > +
> > +
> > +blade.indexerpostprocessor-1.0.0.jar
> > +*******************************************************
> > +
> > +
> > +blade.jsf.portlet.ds-1.0.0.jar
> > +*******************************************************
> > +CHANGED    MANIFEST   <manifest>
> > + CHANGED    HEADER     Import-Package
> > +  CHANGED    CLAUSE     javax.servlet
> > +   ADDED      PARAMETER  version:[3.0,4)
>
> @rotty3000 <https://github.com/rotty3000> are we sending any import
> macros in our liferay-gradle plugin? i guess bndtools may send some default
> import macros, like [=,+)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/gamerson/liferay-blade-samples/pull/12/files/a49bc2bb47960dd416755f6fad63a68032d58c93#r59929143>
>



-- 
*Raymond Augé* <http://www.liferay.com/web/raymond.auge/profile>
 (@rotty3000)
Senior Software Architect *Liferay, Inc.* <http://www.liferay.com>
 (@Liferay)
Board Member & EEG Co-Chair, OSGi Alliance <http://osgi.org> (@OSGiAlliance)